webapp/public/index.html

```diff
@@ -2,14 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <!--Twitter, CC BY 4.0 <https://creativecommons.org/licenses/by/4.0>, via Wikimedia Commons-->
+    <link rel="icon" type="image/svg+xml" href="%PUBLIC_URL%/assets/images/favicon.svg">
```

Comment:
Do you also see this line of html emphasized with a transparentish yellowish overlay here in github? Is it something we have done? Is it the way my browser is rendering this page? Or github?

Reply:
Yes, you're right. I think this is how Github renders HTML comments.